src/Microsoft.ServiceFabric.Services.Remoting/V2/Runtime/ServiceRemotingMessageDispatcher.cs

```diff
@@ -39,11 +41,15 @@ public class ServiceRemotingMessageDispatcher : IServiceRemotingMessageHandler,
         /// <param name="serviceContext">The service fabric service context.</param>
         /// <param name="serviceImplementation">Object that implements the speciifed remoted interfaces.</param>
         /// <param name="serviceRemotingMessageBodyFactory">The factory that will be used by the dispatcher to create response message bodies.</param>
+        /// <param name="activitySourceName">Name of the ActivitySource to be used for tracing</param>
+        /// <param name="activitySourceVersion">Version of ActivitySource to be used for tracing</param>
         public ServiceRemotingMessageDispatcher(
             IEnumerable<Type> remotingTypes,
             ServiceContext serviceContext,
             object serviceImplementation,
-            IServiceRemotingMessageBodyFactory serviceRemotingMessageBodyFactory = null)
+            IServiceRemotingMessageBodyFactory serviceRemotingMessageBodyFactory = null,
+            string activitySourceName = "Microsoft.ServiceFabric.ActivitySource",

```

Comment:
Microsoft.ServiceFabric.Services.Remoting.ActivitySource suits better as this is specific to service remoting calls only?

Reply:
Made the change